devicon.json

```
@@ -9300,6 +9300,29 @@
             }
         ]
     },
+    {
+        "name": "tableau",
+        "altnames": [],
+        "tags": [
+            "data visualization",
+            "software",
+            "business intelligence"
```

Looks like the project is open-source as well: https://github.com/tableau.
```suggestion
            "business intelligence",
            "open-source"
```